track/cptvtrackextractor.py

```diff
@@ -53,7 +48,7 @@ def __init__(self, config, tracker_config):
             os.path.join(self.config.tracks_folder, "dataset.hdf5")
         )
 
-        self.worker_pool_init = init_workers
+        self.worker_pool_init = None
```

Comment:
can you also get rid of the worker_pool_init stuff/worker_pool_args stuff now that it's no longer needed?